### PR TITLE
RFC: Process for repository ownership changes

### PR DESCRIPTION
**Pull request by emalm**
[Link to readable presentation](https://github.com/cloudfoundry/community/blob/rfc-repository-ownership/toc/rfc/rfc-draft-repository-ownership.md)

### CODE REVIEW

**Comment by emalm** [comment]
Also pinging the current WG leads for feedback on this RFC: @ameowlia @rkoster @beyhan @Gerg @pivotal-marcela-campo @paulcwarren @gcapizzi @georgethebeatle @jochenehret

File: toc/rfc/rfc-draft-repository-ownership.md
Comment (stephanme) [inline toc/rfc/rfc-draft-repository-ownership.md:54]: Do we need issues or can we use PRs only?
I would add that the PR should modify the WG charter documents (= not the technical cloudfoundry.yml which is eventually generated from the charters).

Proposed changes to repository ownership SHOULD be submitted for consideration as a pull request on the Working Group charter documents in the Cloud Foundry community repository ...

**Review by stephanme — CHANGES_REQUESTED** [review]
Minor clarification that the PR should edit the WG charter documents because we had a similar discussion recently for approvers.

LGTM otherwise. I used this RFC already for #182

File: toc/rfc/rfc-draft-repository-ownership.md
Comment (stephanme) [reply toc/rfc/rfc-draft-repository-ownership.md]: Looks good now, thanks.

File: toc/rfc/rfc-draft-repository-ownership.md
Comment (stephanme) [inline toc/rfc/rfc-draft-repository-ownership.md:30]: I think we should list the TOC owned repos e.g. in https://github.com/cloudfoundry/community/blob/main/toc/TOC.md in the same format as in WG charters (or in a separate WG charter like document). This makes the list explicit and allows to apply the same/similar processes and automation.
So far we talk only about this community repo, isn't it?

Comment (emalm) [reply toc/rfc/rfc-draft-repository-ownership.md]: Sure, that makes sense. I think the TOC.md file counts as the charter document for the TOC (apart from the real definition of the TOC scope in the Foundation charter document), so that seems like a good place to list both sets of repositories. Pushed https://github.com/cloudfoundry/community/pull/251/commits/8e86f682d29040125a12ce4c272f1a6a5a0f1412 to adjust this language.

**Comment by ameowlia** [comment]
LGTM

**Comment by emalm** [comment]
@LeePorte @stephanme @dsboulder @loewenstein I'm making a motion for the Final Comment Period on this RFC to start. Please reply or react to this comment to vote asynchronously, and we can also discuss and vote in the upcoming TOC meeting.

**Comment by emalm** [comment]
@LeePorte @stephanme @dsboulder As a reminder, please vote on [the motion to start the week-long Final Comment Period](https://github.com/cloudfoundry/community/pull/251#issuecomment-1109854008) for this RFC. Only @loewenstein and I have voted to approve the motion so far.

**Review by LeePorte — APPROVED** [review]

**Comment by emalm** [comment]
We now have a quorum of TOC approvals to start the Final Comment Period. I'm re-requesting review from @cloudfoundry/toc for approvals to accept the RFC. My vote is to approve, since I can't request myself to review my own PR.

Everyone else, if you've been waiting to weigh in, now's your chance!

**Review by LeePorte — APPROVED** [review]

**Comment by emalm** [comment]
By the way, the week-long Final Comment Period ends on May 10th, 7:30 am PDT / 10:30 am EDT / 15:30 pm BST / 16:30 pm CEST.

**Review by Gerg — APPROVED** [review]

**Review by stephanme — APPROVED** [review]

**Pull request merged**
